.github/workflows/test.yml

```diff
@@ -241,6 +248,83 @@ jobs:
         gitlab-org: ${{ secrets.GITLAB_ORG }}
         artifacts-base-dir: "/tmp/gitops-test"
 
+  notify-failure:
+    name: Send failure notifications
+    runs-on: ubuntu-latest
+    needs:
+      - test
+      - smoke-tests
+      - smoke-tests-long
+      - acceptance-tests-0
+      - acceptance-tests-1
+      - acceptance-tests-2
+      - acceptance-tests-3
+      - ui-tests
+    if: ${{ failure() }}
```

Comment:
:+1: Didn't know we could do this. Branching in `yaml`, what a time to be alive.